packages/java/parser-jvm-plugin-transfertypes/src/main/java/dev/hilla/parser/plugins/transfertypes/UUIDReplacer.java

```diff
@@ -0,0 +1,21 @@
+package dev.hilla.parser.plugins.transfertypes;
+
+import static dev.hilla.parser.plugins.transfertypes.TransferTypesPluginUtils.createReplacer;
+
+import java.util.UUID;
+
+import dev.hilla.parser.core.ClassMappers;
+
+final class UUIDReplacer implements Replacer {
+    private ClassMappers classMappers;
+
+    @Override
+    public void process() {
+        classMappers.add(createReplacer(UUID.class, String.class));
```

Comment:
Now that the `Replacer` interface is introduced, it’s a bit confusing here that `createReplacer` returns something else. Should we rename `createReplacer` to `createMapper`?